pytorch_lightning/trainer/connectors/data_connector.py

```diff
@@ -30,6 +30,12 @@ def on_trainer_init(self, check_val_every_n_epoch, reload_dataloaders_every_epoc
         self.trainer.datamodule = None
```

Comment:
could you add typehints for the arguments in `on_trainer_init`?

Reply:
Done!